rss-client/src/app/Account/components/account-settings-page/account-settings-page.component.ts

```diff
@@ -131,15 +135,31 @@ export class AccountSettingsPageComponent implements OnInit {
   }
 
   createAccount(event) {
+    let index = null;
     if (event == 'Eval') {
       let num = this.accounts[1].accTypeId;
       this.myAccount.accTypeId = num;
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 1;
     } else if (event == 'Bug') {
       this.myAccount.accTypeId = this.accounts[0].accTypeId;
       this.myAccount.userId = this.userservice.userPersistance().userId;
+      index = 0;
+    }
+    else if (event == 'Overflow'){
+      if(this.accounts[2] == undefined || this.accounts[2] == null){
+        alert("There is not a Rev Overflow account linked with your user");
+      }
+      else{
+        this.myAccount.accTypeId = this.accounts[2].accTypeId;
+        this.myAccount.userId = this.userservice.userPersistance().userId;
+      }
+      index = 2;
+    }
+    if(this.accounts[index] !== undefined && !this.accounts[index] !== null){
+      console.log("hit");
+      this.accountService.createAccount(this.myAccount).subscribe((res) => {});
```

Comment:
What is supposed to happen if this if statement fails. Is it okay that nothing happens or should there be an alert for the user?